### PR TITLE
Documentation for the Domain Manager standard

### PR DESCRIPTION
**Pull request by markus-hentsch**
Closes SovereignCloudStack/issues#385

### CODE REVIEW

File: docs/05-iam/domain-manager-setup-and-usage.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+This role offers a properly domain-scoped permission set to manage users, groups, projects and role assignments within a domain.
+Its intended use case is to offer extensive identity management self-service capabilities to tenants mapped to a domain.
+
+This guide will explain setup, configuration and usage of the SCS Domain Manager standard.
+
+### Warning regarding the exposure of domain names
+
+Due to architectural limitations currently existing in OpenStack Keystone, assigning the Domain Manager role to users will **enable them to see the IDs and names of all existing domains**.
+This includes domains other than their own, meaning that other tenant's identities might be exposed depending on the relation between them and the name of their domain.
+CSPs aiming to appoint Domain Manager users must be aware of this limitation and should exclusively **use pseudonymized domain names across the whole infrastructure**.
```
Comment (JuanPTM) [inline docs/05-iam/domain-manager-setup-and-usage.md:19]: Maybe a link to the standard for naming convention can help here. 
https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0301-v1-naming-conventions.md

Comment (markus-hentsch) [reply docs/05-iam/domain-manager-setup-and-usage.md:19]: Good point! I've added this to the text. Thank you!

**Review by josephineSei — APPROVED** [review]
I tested this guide and was able to set everything up. I only got confused over the current policy. But that is part of other PRs

**Comment by josephineSei** [comment]
@JuanPTM could you give me a hint on how to resolve the merge conflict. And it would be good to have another one check the document itself. Because markus is currently not around to resolve or merge this PR.

**Review by maxwolfs — APPROVED** [review]
LGTM – Thanks!

**Pull request merged**